src/librustdoc/html/static/rustdoc.css

```diff
@@ -1141,36 +1141,53 @@ kbd {
 	border: solid 1px;
 	border-radius: 3px;
 	box-shadow: inset 0 -1px 0;
+	cursor: default;
 }
 
-#theme-picker {
+.theme-picker {
 	position: absolute;
 	left: 211px;
-	top: 17px;
+	top: 19px;
+}
+
+#theme-picker {
 	padding: 4px;
+	width: 27px;
+	height: 29px;
 	border: 1px solid;
 	border-radius: 3px;
 	cursor: pointer;
 }
 
+#theme-picker:hover {
```

Comment:
`:focus`?

Reply:
Should be in `main.css` actually. Strange...